zstackctl/zstackctl/zstacklib.py

```
@@ -33,10 +34,152 @@
+    ])
+
+
+def get_properties_file_name(zstack_home=None, app_name=None):
+    classes_dir = get_classes_dir(zstack_home)
+    candidate_names = get_candidate_properties_file_names(app_name)
+    cache_key = (classes_dir, tuple(candidate_names))
+    cached = PROPERTIES_FILE_NAME_CACHE.get(cache_key, _CACHE_MISS)
+    if cached is not _CACHE_MISS:
+        return cached
+
+    app_config_path, properties_file_name = get_app_config_details(classes_dir)
```

Ruff (0.15.5)

[warning] Unpacked variable `app_config_path` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)